orchestration/dagster_orchestration/hca_orchestration/pipelines/cut_snapshot.py

```diff
@@ -89,13 +89,14 @@
 
 
 @success_hook(
-    required_resource_keys={'slack', 'snapshot_config', 'dagit_config'}
+    required_resource_keys={'slack', 'snapshot_config', 'dagit_config', 'hca_manage_config'}
 )
 def snapshot_start_notification(context: HookContext) -> None:
     context.log.info(f"Solid output = {context.solid_output_values}")
     job_id = context.solid_output_values["result"]
     kvs = {
         "Snapshot name": context.resources.snapshot_config.snapshot_name,
+        "Project ID": context.resources.hca_manage_config.google_project_name,
```

Comment:
⛏️ Can you call this `Google Project ID` to avoid any ambiguity with HCA project id's?